lab-davide/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "lab-davide",
+  "version": "1.0.0",
+  "description": "",
+  "main": "hello.js",
+  "directories": {
+    "test": "test"
```

Comment:
Instead of "test" use "mocha", what this does is specify what the command npm test will perform, which in your case here, you'd want to be to run mocha.  So line 7 should look like this:
"test": "mocha"